clang/test/Analysis/std-variant-checker.cpp

```diff
@@ -58,6 +58,14 @@ void wontConfuseStdGets() {
 //----------------------------------------------------------------------------//
 // std::get
 //----------------------------------------------------------------------------//
+void stdGetType2() {
```

Comment:
(Style nit: if this is `2`, move it below the other one?)

Reply:
Oh, this test is not relevant. It is basically the same as the one after it. I was just doing some debugging to see how CSA handles different kinds of object constructions and forgot to remove it. I will remove it.